tee/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(sgx)
```

Comment:
I think this tee folder's naming here is anbiguous because it includes the sgx TEE.

Reply:
sgx is too specific. So folder tee is more common to user